src/Classes/Canvas.ts

```diff
@@ -61,6 +62,19 @@ export class Canvas {
 
                 /* density particles enabled */
                 container.densityAutoParticles();
+
+                if (options.polygon.type !== PolygonMaskType.none) {
+                    if (container.polygon.redrawTimeout) {
+                        clearTimeout(container.polygon.redrawTimeout);
+                    }
+
+                    container.polygon.redrawTimeout = setTimeout(async function () {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature to have a typedef](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5042590)